src/components/sbb-radio-button-group/sbb-radio-button-group.e2e.ts

```diff
@@ -0,0 +1,117 @@
+import { E2EPage, newE2EPage, E2EElement } from '@stencil/core/testing';
```

Comment:
if possible: a test about disabled state from radio-button-group which should overwrite the radio-button disabled state and then after removing, it should then take the disabled from radio-button again.